planning/behavior_path_start_planner_module/README.md

```diff
@@ -2,17 +2,23 @@
 
 ## Purpose / Role
 
-The Start Planner module is designed to generate a path from the current ego position to the driving lane, avoiding static obstacles and implementing safety checks against dynamic obstacles. (Note: The feature of safety checks against dynamic obstacles is currently a work in progress.)
-This module is activated when a new route is received.
```

Comment:
This is not needed?

Reply:
changed expression in this PR from
```
The Start Planner module is designed to generate a path from the current ego position to the driving lane, avoiding static obstacles and implementing safety checks against dynamic obstacles. (Note: The feature of safety checks against dynamic obstacles is currently a work in progress.)
```
to
```
The Start Planner module is designed to generate a path from the current ego position to the driving lane, avoiding static obstacles and stopping in response to dynamic obstacles when a collision is detected.
```

And for the following description is not correct in my understanding :thinking: 
```
This module is activated when a new route is received.
```
activated condition is not only receiving new route?

Reply:
I thought the tivated condition is only receiving new route. What are other conditions?
(I can not check the code details now🙇)

Reply:
memo:
ex) even if reciving new route, the ego deviates from center line the module is not executed.

Reply:
will update to add acivation condition section